container/dockerd-entrypoint.sh

```diff
@@ -2,14 +2,26 @@
 set -e
 set -x
 
+export IFS=' '
+MMS_NUM_WORKER=1
+MMS_NUM_THREAD_PER_WORKER=`nproc`
 if [[ "$1" = "serve" ]]; then
+    shift 1
     cp -v /opt/ml/model/* /home/model-server/model
     cp -v /home/model-server/neo_template.py /home/model-server/model
     model-archiver --handler neo_template:predict --model-name neomodel --model-path /home/model-server/model -f --export-path /home/model-server
     mv /home/model-server/neomodel.mar /home/model-server/model
 
-    shift 1
-    mxnet-model-server --start --mms-config config.properties
+    if [ ! -z "${MMS_NUM_WORKER}" ]
+    then
```

Comment:
line 8 has `if` and `then` on the same line. Can you do the same on line 15?

Reply:
sure